docs/en/authenticators.rst

```diff
@@ -171,6 +173,41 @@ In your ``UsersController``::
         $this->viewBuilder()->setOption('serialize', 'json');
     }
 
+Using a JWKS fetched from an external JWKS endpoint is supported as well::
+
+    // Application.php
+    public function getAuthenticationService(ServerRequestInterface $request): AuthenticationServiceInterface
+    {
+        $service = new AuthenticationService();
+        // ...
+        $service->loadIdentifier('Authentication.JwtSubject');
+
+        $jwksUrl = 'https://appleid.apple.com/auth/keys';
+
+        // Set of keys. The "keys" key is required. Additionally keys require a "alg" key.
+        // Add it manually to your JWK array if it doesn't already exist.
+        $jsonWebKeySet = Cache::remember('jwks-' . md5($jwksUrl), function () use ($jwksUrl) {
+            $http = new Client();
+            $response = $http->get($jwksUrl);
+            return $response->getJson();
+        });
+
+        $service->loadAuthenticator('Authentication.Jwt', [
+            'jsonWebKeySet' => $jsonWebKeySet,
+            'returnPayload' => false
+        ]);
+    }
+
+The JWKS resource will return the same set of keys most of the time.
+Applications should cache these resources, but they also need to be
+prepared to handle signing key rotations.
+
+.. warning::
+
+    An application would need to decide on a cache lifetime that strikes a good
+    balance between performance and security, particularly for the emergency
+    case where a private key is compromised.
```

Comment:
```suggestion
    Applications need pick a cache lifetime that balances performance and security.
    This is particularly important in situations where a private key is compromised.
```

Reply:
@othercorey - isn't it "Applications need **to** pick"

Reply:
Yes, missing `to`.